viewer/app/styles/viewer.css

```diff
@@ -77,13 +87,25 @@ body {
 }
 .viewer-placeholder__heading {
   font-weight: 300;
-  margin: 0;
+  margin: 0 0 24px;
   line-height: 32px;
 }
 .viewer-placeholder__help {
-  margin-top: 12px;
   line-height: 1.6;
 }
+
+.viewer-placeholder__file-button {
+  border: 0;
+  background: 0;
+  display: inline;
+  padding: 0;
+  color: #0000ee;
+  color: -webkit-link;
```

Comment:
spruce up my CSS please... won't the second `color:` just negate the first, even in a browser that does not recognize `-webkit-link`?

Reply:
second color negates the first only IF the second is supported.

but you made me doublecheck. thankfully, all good:
![image](https://user-images.githubusercontent.com/39191/222838789-bdbbc145-8210-4de3-83a8-f5a62b96c7d8.png)


------------

where this kinda thing is a problem is if you're using a (psuedo)selector that has partial support...
then the entire ruleset is thrown out, for example:

```css
h1, a:-webkit-any-link {
  background: red;
}
```

<img width="1687" alt="image" src="https://user-images.githubusercontent.com/39191/222839581-c49bc351-72b9-44f9-86c9-189335656b6d.png">

but property value support fallback is really forgiving, afaik.